libs/sdk-ui-kit/src/List/InsightListItemDate.tsx

```diff
@@ -1,17 +1,24 @@
 // (C) 2007-2020 GoodData Corporation
 import React from "react";
+import { utcToZonedTime } from "date-fns-tz";
+import { subDays, format, parseISO } from "date-fns";
```

Comment:
Please import individual parts (similarly to lodash) to improve tree-shakability. For extra awesomeness, please add dep-crusire rule for this (there already is one for lodash).

Reply:
Fixed, including the dependency cruiser rules (for both `date-fns` and `date-fns-tz`).

Reply:
Awesome! 👍